src/components/test/Spinner-test.js

```diff
@@ -12,6 +12,11 @@ describe('Spinner', () => {
     assert.isTrue(wrapper.find('SvgIcon.Hyp-Spinner--medium').exists());
   });
 
+  it('uses the registered `hyp-spinner` icon', () => {
+    const wrapper = createSpinner();
+    assert.equal(wrapper.find('SvgIcon').props().name, 'hyp-spinner');
+  });
+
```

Comment:
Work tracked for follow up: Don't register any icons as part of test bootstrapping; tests should fail if a component references an icon it doesn't register itself, even if a test is not written for the icon itself (i.e. a component will fail to render in test because the icon will be missing).